modules/stanford_bean_types_featured/scss/stanford_bean_types_featured.scss

```diff
@@ -482,6 +482,11 @@ div[class*="featured"] {
       top: 65%;
     }
 
+    // TARGET CHROME
+    @media screen and (-webkit-min-device-pixel-ratio:0) and (min-resolution:.001dpcm) {
```

Comment:
I don't think we need to target just Chrome. I tested the CSS injector file on Firefox, Chrome, and Safari. It worked on all three.

Reply:
sounds good
I'll make that change.

Reply:
@cjwest I tested on my local the injector, but Safari didn't look right at all. I think taking css browser hacks out and compromising on the top to 60% looks good in the browsers.

Reply:
@minorwm,
Safari is not cooperating. Check this out in Safari:
https://eng-dev.stanford.edu/test-banner-images